src/open_inwoner/react/components/SelectComboBox/SelectComboBox.tsx

```diff
@@ -0,0 +1,369 @@
+import {
+  FC,
+  useState,
+  useRef,
+  useEffect,
+  ChangeEvent,
+  KeyboardEvent,
+} from 'react'
+import { useIntl } from 'react-intl'
+import clsx from 'clsx'
+import { MaterialIcon } from '../MaterialIcons/MaterialIcons'
```

Comment:
I can't remember for sure, but didn't we alias `@` so we can do something like:


```suggestion
import { MaterialIcon } from '@/components/MaterialIcons/MaterialIcons'
```

Reply:
Yes but a bit different,  "@react/" and   "@webcomponents/".